lib/rules/accessor-pairs.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Rule to flag wrapping non-iife in parens
+ * @author Gyandeep Singh
+ * @copyright 2015 Gyandeep Singh. All rights reserved.
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+    var config = context.options[0] || {};
+    var checkGetWithNoSet = config.getWithNoSet === true;
+
+    /**
+     * Checks a object expression to see if it has setter and getter both present or none.
+     * @param {ASTNode} node The node to check.
+     * @returns {void}
+     * @private
+     */
+    function checkLonelySetGet(node) {
+        var isSetPresent = node.properties.some(function(prop) {
+            if (prop.kind === "set") {
```

Comment:
Inside the rule whenever I listen for `ObjectPattern` or `ObjectPattern:exit`, the attached function never gets called.
Has anyone faced this issue before or am I missing something here?